helm/istio/charts/tracing/templates/jaeger-all-in-one.yaml

```diff
@@ -0,0 +1,68 @@
+{{ if and (eq .Values.provider "jaeger") (eq .Values.jaeger.template "all-in-one") }}
+apiVersion: jaegertracing.io/v1
+kind: "Jaeger"
+metadata:
+  name: "jaeger"
+  namespace: {{ .Release.Namespace }}
+  labels:
+    chart: {{ template "tracing.chart" . }}
+    heritage: {{ .Release.Service }}
+    release: {{ .Release.Name }}
+spec:
+  strategy: allInOne
+
+  allInOne:
+    image: {{ .Values.jaeger.hub }}/all-in-one:{{ .Values.jaeger.tag }}
+    options:
+      log-level: debug
+      query:
+        base-path: /jaeger
+    annotations:
+      {{- range $key, $value := .Values.jaeger.annotations }}
+        {{ $key }}: {{ $value | quote }}
+      {{- end }}
+
+  agent:
+    options:
+      collector:
+        host-port: jaeger-collector:14267
+
+  ui:
+    options:
+      dependencies:
+        menuEnabled: false
+      gaTrackingID: UA-000000-2
+      menu:
+        - label: "About Jaeger"
```

Comment:
Would be good to update the menu to be similar to the default one used upstream: https://github.com/jaegertracing/jaeger-operator/blob/master/pkg/strategy/controller.go#L224-L237 - specifically to take into account the "Log Out" menu item.

Reply:
Hey @objectiser, I notice that the link you pasted uses `/oauth/sign_in` for the log out url.  Is that correct?

Reply:
never mind.  works like a charm